posts/test_post_uploading.py

```diff
@@ -16,16 +16,21 @@
 # limitations under the License.
 
 import unittest
+from unittest import mock
 import mongomock
 import app
 
+MOCK_FILE_URL = "the url of an uploaded file"
+MOCK_FILE = mock.MagicMock()
+MOCK_FILE.filename = "the name of a file"
```

Comment:
```python
File = collections.namedtuple('File', 'filename')
MOCK_FILE = File(filename="the name of a file")
```

Doesn't require any docstrings.  😁

Reply:
(or you could instantiate File down where MOCK_FILE is right now.)

Reply:
Done https://github.com/knative-portability/large-events/pull/20/commits/11715c843ba3f11080db72606e658d223bc35adf